src/pages/GivePage/GiveAllocator.tsx

```diff
@@ -104,14 +103,15 @@ export const GiveAllocator = ({
           >
             <MinusCircle css={{ width: iconSize, height: iconSize }} />
           </Button>
-          {(gift.tokens < 1 || disabled) && <ClickTrapperIcon />}
+          {((gift.tokens ?? 0) < 1 || disabled) && <ClickTrapperIcon />}
         </Flex>
         <TextField
           data-testid="tokenCount"
-          value={gift.tokens}
+          value={gift.tokens ?? ''}
           onChange={evt => setGiftTokens(evt.target.value)}
           maxLength={5}
           disabled={disabled}
+          onBlur={() => setGiftTokens(gift.tokens ? String(gift.tokens) : '0')}
```

Comment:
Added this so that when the user loses focus on an empty input field it will automatically populate with a `0` -- otherwise, the input field would stay empty. It's just for a better UX, the maths for GIVEs would work correctly.